plugins/mcreator-localization/lang/texts.properties

```
@@ -1940,6 +1941,16 @@ elementgui.gui.import_overlay_base_texture=Click this to import overlay base tex
 elementgui.gui.overlay_base_texture=Base texture:
 elementgui.gui.edit_component_message=This component can only be repositioned or removed!
 elementgui.gui.edit_component_title=Edit component
+elementgui.item_extension.item=Item:
+elementgui.item_extension.fuel_properties=Custom fuel properties
+elementgui.item_extension.enable_fuel=Make this item a fuel:
+elementgui.item_extension.burn_time=Fuel burn time (power):
+elementgui.item_extension.layer_chance=Chance of success to compost
```

To follow your other strings:
```suggestion
elementgui.item_extension.layer_chance=Chance of success to compost:
```